strapi/strapi
# [bulk publish] Change published label in the modal

### What does it do?

- Change the published label in the status column inside the Selected Entries Modal to "Already Published"

### Why is it needed?

- because it explains better the status

### How to test it?

- Select a bunch of entries in the ListView, some of them published
- click Publish
- you can now see the status "Already Published"

Bug related CS-127
